packages/pigeon/example/README.md

```diff
@@ -5,6 +5,8 @@
 This example gives an overview of how to use Pigeon to call into the
 host-platform from Flutter.
 
+First, follow the [steps](https://pub.dev/packages/pigeon#usage) to set up pigeon for iOS and Android.
+
```

Comment:
I don't know that this is helpful because those steps aren't required to read or use the example.  If we thinking adding that link is a good idea it would be better to say something like:

> For instructions to set up your own Pigeon usage see these steps.

Reply:
That makes sense, added the changes